### PR TITLE
[Merged by Bors] - feat(Cache): include MathlibTest in the cache

### PR DESCRIPTION
**Pull request by grunweg**
---------

Not tested locally, but looks simple enough.

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Cache/IO.lean [hunk 1]
```diff
@@ -132,6 +132,7 @@ private def CacheM.getContext : IO CacheM.Context := do
     ("Mathlib", root),
     ("Archive", root),
     ("Counterexamples", root),
+    ("MathlibTest", root / "MathlibTest"),
```
Comment (adomani) [inline Cache/IO.lean:135]: The other lines that use `root` do not have `/ "MathlibTest"` as well.  Is this correct as is?

Comment (grunweg) [reply Cache/IO.lean:135]: Good question! Probably removing the `MathlibTest` is more correct.
```suggestion
    ("MathlibTest", root),

**Review by adomani — APPROVED** [review]
I agree that this looks just like the previous lines now, but I do not really know what `cache` does, so I would like to see someone else going over this to make sure that it actually works as intended.  My approval is just cargo-culting!  :smile:

**Comment by kim-em** [comment]
bors merge

**Pull request closed without merging**